content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Multi-account and Multi-region Testing"
+weight: 8
+description: >
+  How to test multi-account and multi-region compatibility.
+---
+
+LocalStack supports multi accounts and regions compatibility. 
```

Comment:
nit: I think we should stick to the terms combined with a dash: `multi-account`, `multi-region`, `cross-account`, `inter-service-communication`. What's your opinion? Are there any guidelines or best practices on this, @HarshCasper?
```suggestion
LocalStack supports multi-accounts and multi-regions compatibility.
```